cranelift/filetests/filetests/runtests/br_jumptable.clif

```diff
@@ -0,0 +1,48 @@
+test interpret
+
+
+function %indirect_jump_table_br_i64(i64) -> i64 {
```

Comment:
Both of these tests don't work in the actual compiler, but I'm not sure if its a bug.

It looks like we convert from `br_tables` into `indirect_jump_table_br` at some point during compilation. But do we support these instructions directly?

The crash that i'm seeing is the following:
```
thread 'worker #0' panicked at 'assertion failed: self.f.dfg[inst].opcode() == Opcode::BrTable', cranelift\codegen\src\machinst\lower.rs:901:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
FAIL .\filetests\filetests\runtests\br_jumptable.clif: panicked in worker #0: assertion failed: self.f.dfg[inst].opcode() == Opcode::BrTable
1 tests
Error: 1 failure
```

Reply:
These instructions are only used by the old x86 backend as part of a lowering from `br_table`. The new x64 backend uses `br_table` directly.

Reply:
Oh, I didn't realize that, I thought that they were still valid because this [test](https://github.com/bytecodealliance/wasmtime/blob/main/cranelift/filetests/filetests/licm/rewrite-jump-table.clif) which targets aarch64 still uses them.

I'll remove the implementation for this from the PR

Reply:
By the way, are `brif` and `brff` also replaced by `br_icmp`/`fcmp`?

Reply:
I don't think that is decided yet.